projects/storefrontapp-e2e-cypress/cypress.ci.2005.json

```diff
@@ -5,6 +5,9 @@
   "projectId": "k3nmep",
   "numTestsKeptInMemory": 1,
   "video": true,
+  "retries": {
+    "runMode": 3
```

Comment:
With 3 retries you are saying you want it to run a maximum of 4 times is this what you want? or should it be `runMode: 2` for a total of 3 runs?